src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -327,16 +323,16 @@
              ;; method redact is responding with "code" for 401.  TODO: standardize and remove this extra logic
              status-code (or status-code code)]
          (if-let [expected-msg (get-in props [:expect status-code])]
-           [:div {}
+           [:div {:style {:display "flex" :alignItems "center"}}
             [:span {:style {:paddingRight "1ex"}}
-             (icons/font-icon {:style {:color (:exception-red style/colors)}}
-               :status-warning-triangle)]
+             (icons/icon {:style {:color (:exception-red style/colors)}}
+                         :warning-triangle)]
             (str "Error: " expected-msg)]
            [:div {:style {:textAlign "initial"}}
-            [:div {}
+            [:div {:style {:display "flex" :alignItems "center"}}
```

Comment:
Yeah, a lot of these cases are just for sticking an icon next to text.  There could be a util for this.